backend/src/test/java/com/bakdata/conquery/integration/sql/SqlIntegrationTestSpec.java

```diff
@@ -92,6 +99,8 @@ public void executeTest(SqlStandaloneSupport support) throws IOException {
 
 		SqlExecutionResult result = managedQuery.getResult();
 		List<EntityResult> resultCsv = result.getTable();
+		// TODO (ask awildturtok) can we use printer here?
```

Comment:
Ich wollte eigentlich den `ResultCsvProcessor` nutzen um den Vergleich von actual und expected CSV so ähnlich wie im `AbstractQueryEngineTest` zu machen. Allerdings bekomme ich immer Probleme, dass anscheinend irgendwelche Config Settings in Bezug auf User nicht richtig gesetzt sind. Da brauche ich dann nochmal deine Hilfe.

Reply:
uff, kannst du mir die Stacktrace dfür schicken? Ist leider nicht ganz mein Thema, aber ich werde es mir anschauen